servers/assets/index.js

```diff
@@ -1,3 +1,25 @@
 require('@orbiting/backend-modules-env').config()
 const server = require('./server')
-server.run()
+const regiment = require('regiment')
+
+const {
+  CLUSTER,
+  WEB_CONCURRENCY = 1,
+  WEB_MEMORY = 512
+} = process.env
+
+if (CLUSTER) {
+  server.addMiddleware(
+    (server) =>
+      server.use(regiment.middleware.MemoryFootprint(WEB_MEMORY))
+  )
+  regiment(
+    server.start,
+    {
+      numWorkers: WEB_CONCURRENCY,
+      deadline: 30000
```

Comment:
Annotate units here.

I find this pattern pretty neat, assuming `deadline` is in miliseconds:

```
 deadline: 30 * 1000 // 30 seconds
```

Or if it were 5 minutes:

```
 deadline: 5 * 60 * 1000
```

Or 9 hours...

```
 deadline: 9 * 60 * 60 * 1000
```